lambda/shared/src/main/scala/feral/lambda/events/ApiGatewayProxyResult.scala

```diff
@@ -17,26 +17,35 @@
 package feral.lambda.events
 
 import io.circe.Encoder
+import org.typelevel.ci.CIString
 
 sealed abstract class ApiGatewayProxyResult {
   def statusCode: Int
+  def headers: Map[CIString, String]
   def body: String
   def isBase64Encoded: Boolean
 }
 
 object ApiGatewayProxyResult {
 
-  def apply(statusCode: Int, body: String, isBase64Encoded: Boolean): ApiGatewayProxyResult =
-    new Impl(statusCode, body, isBase64Encoded)
```

Comment:
Woops, good catch! Can you keep a `@deprecated` version of the old constructor too, for backwards-compatibility? Thank you!